build.zig

```diff
@@ -2,9 +2,14 @@ const std = @import("std");
 const builtin = @import("builtin");
 const CrossTarget = std.zig.CrossTarget;
 const Mode = std.builtin.Mode;
+const file_or_directory_exists = @import("src/clients/shutil.zig").file_or_directory_exists;
```

Comment:
If this is the only function you're pulling from that file and it's only three lines, maybe we should just copy the code instead?